### PR TITLE
CI with GitHub actions

### PR DESCRIPTION
**Pull request by gordonmessmer**

### CODE REVIEW

**Comment by gordonmessmer** [comment]
I haven't worked with GitHub Actions yet, but this seems reasonably straightforward.

Example job log: https://github.com/gordonmessmer/python-rpm-packaging/runs/2416955119?check_suite_focus=true

Fixes issue #6

File: .github/workflows/pytest.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+name: Run pytest
+
+on:
+  push:
+    branches: [ $default-branch ]
```
Comment (hroncok) [inline .github/workflows/pytest.yml]:
```suggestion
```

File: .github/workflows/pytest.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+name: Run pytest
```
Comment (hroncok) [inline .github/workflows/pytest.yml]:
```suggestion
name: Run pytest with latest Fedora packages
```

Comment (hroncok) [reply .github/workflows/pytest.yml]: We'll likely test with other stuff later

File: .github/workflows/pytest.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  test:
+    runs-on: ubuntu-latest
+    container:
+      image: fedora:latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+      - name: Install dependencies
+        run: |
+          dnf install -y python3 python3-pip python3-setuptools python3-pytest python3-pyyaml python3-wheel python3-poetry
```
Comment (hroncok) [inline .github/workflows/pytest.yml]:
```suggestion
          dnf install -y python3-devel python3-pip python3-setuptools python3-pytest python3-pyyaml python3-wheel python3-poetry
```

**Review by hroncok — APPROVED** [review]
Looking good! Couple suggestions added. I have more ideas but we can iterate on them after merging.

Thank you!

**Pull request merged**